store/etcd/rulesets.go

```diff
@@ -36,6 +35,52 @@ func computeLimit(l int) int {
 	return l
 }
 
+// Get returns the ruleset related to the given path. By default, it returns the latest one.
+// It returns the related ruleset version if it's specified.
+func (s *RulesetService) Get(ctx context.Context, path, version string) (*store.RulesetEntry, error) {
+	var (
+		entry *store.RulesetEntry
+		err   error
+	)
+
+	if version == "" {
+		entry, err = s.Latest(ctx, path)
+	} else {
+		entry, err = s.OneByVersion(ctx, path, version)
+	}
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := s.Client.KV.Get(ctx, s.versionsPath(path))
```

Comment:
:tulip: Line 55 to 66, and lines 68 to 79 differ only in the path the call to `s.Client.KV.Get` receives and a noun in the error messages.  This could be wrapped up in a function.  Whilst there might not be any other case of reuse, it again makes the flow of this function clearer, and potential stops bug duplication.